rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -309,6 +336,8 @@ bool Player::play_next()
     return false;
   }
 
+  RCLCPP_INFO_STREAM(this->get_logger(), "Playing next message.");
```

Comment:
While we're adding these infos - maybe a `WARN` for the "not paused" case above? `Play next requested but not paused` something like that